service/model/release.go

```diff
@@ -51,14 +56,6 @@ type Release struct {
 	Attachments  []ReleaseAttachment
 }
 
-type ReleaseAttachment struct {
```

Comment:
The struct was not removed; it was just moved to a lower part of the file. See the changes below.